src/Application/Features/Employees/Handlers/SeedEmployeeCommandHandler.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Net.Http.Json;
+using MediatR;
+using MRA.AssetsManagement.Domain.Entities;
+public class SeedEmployeeCommand : IRequest<string>
```

Comment:
We don't need this handler:
instead create EmployeeEntitySeeder.cs class to the infrastructure/Data/Seeder/Entities

see other examples